tools/debug_tools/src/debug_ui.rs

```diff
@@ -98,6 +59,12 @@ pub fn change_infotext(
             }
         }
 
-        text.sections[0].value = format!(" {fps:.1} fps, {frame_time:.3} ms/frame",);
+        if bools.dev_mode {
+            if bools.show_fps_info {
+                text.sections[0].value = format!(" {fps:.1} fps, {frame_time:.3} ms/frame");
+            } else if !bools.show_fps_info {
+                text.sections[0].value = String::new();
+            }
```

Comment:
I think this can be simplified:

```suggestion
            text.sections[0].value = if bools.show_fps_info {
                 format!(" {fps:.1} fps, {frame_time:.3} ms/frame");
            } else {
                String::new();
            };
```